.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  BUILD_PROFILE: Release
+
+jobs:
+  build:
+    runs-on: windows-2022
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup MSBuild
+        uses: microsoft/setup-msbuild@v1.1
+      - name: Build
+        run: msbuild /p:Configuration=${{ env.BUILD_PROFILE }}
+      - name: Extract Short Commit Hash
+        id: extract
+        shell: bash
+        run: echo ::set-output name=commit::`git rev-parse --short HEAD`
+      - name: Upload Build Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: NorthstarLauncher-${{ steps.extract.outputs.commit }}
+          path: |
+            x64/${{ env.BUILD_PROFILE }}/*.dll
+            x64/${{ env.BUILD_PROFILE }}/*.exe
+            x64/${{ env.BUILD_PROFILE }}/*.pdb
```

```suggestion
            x64/${{ env.BUILD_PROFILE }}/*.pdb
            x64/${{ env.BUILD_PROFILE }}/*.txt
```

Can you also add this to include files generated by https://github.com/R2Northstar/NorthstarLauncher/pull/37 ?